src/authentication/requestPasswordReset.ts

```diff
@@ -1,10 +1,20 @@
 import { createClientComponentClient } from "@supabase/auth-helpers-nextjs";
 import { logErrorReturnLogId } from "@/logger/logger";
+import { AuthError } from "@supabase/gotrue-js";
 
 interface RequestPasswordResetResponse {
     errorMessage: string | null;
 }
 
+function formatErrorMessage(error: AuthError): string {
+    let errorMessage = error.message;
+    // HTTP 429 is "too many requests". Usually means email rate limit has been exceeded.
+    if (error.status === 429) {
+        errorMessage += ".\nPlease contact admin for help.\nhelp@foodbankapp.org";
```

Comment:
Should we save the email into a variable so that it can be changed easily?

Reply:
Done! Wasn't sure where the best place to put the variable was so I left it in the same function for now :)